### PR TITLE
cleanup and last touches on floating action button and icon components

### PR DESCRIPTION
**Pull request by szutrifork**
Cleanup and small fixes that are missing from last week

### CODE REVIEW

File: src/app/showcase/icon-showcase/icon-showcase.component.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.performance {
+    margin-top: 20px;
```
Comment (jakobe) [inline src/app/showcase/icon-showcase/icon-showcase.component.scss]:
```suggestion
    margin-top: 20px;
    color: get-color('kirby-grey-8');
```

File: src/app/showcase/icon-showcase/icon-showcase.component.html [hunk 2]
```diff
@@ -3,4 +3,16 @@
   <kirby-html-viewer [html]="exampleHtml"></kirby-html-viewer>
   <h4>Properties:</h4>
   <kirby-showcase-properties [properties]="properties"></kirby-showcase-properties>
-</div>
\ No newline at end of file
+
+  <div class="performance">
+    <div class="title">
+      <ion-icon name="information-circle"></ion-icon>
```
Comment (jakobe) [inline src/app/showcase/icon-showcase/icon-showcase.component.html]: Can we make the icon blue? :) ℹ️

File: src/kirby/components/floating-action-button/floating-action-button.component.html [hunk 3]
```diff
@@ -1,3 +1,3 @@
-<ion-fab-button class="fab {{ showShadow ? '' : 'no-shadow' }}" [disabled]="disabled">
+<ion-fab-button class="fab {{ !showShadow || disabled ? 'no-shadow' : '' }} {{ disabled ? 'disabled' : '' }}" [disabled]="disabled">
```
Comment (jakobe) [inline src/kirby/components/floating-action-button/floating-action-button.component.html]: Could be simplified, by using a `HostBinding` in the component instead:
```ts
@HostBinding('class.has-shadow')
@Input() showShadow?: boolean = true;
```

And then in the scss:
```css
ion-fab-button,
ion-fab-button('[disabled]') {
  --box-shadow: 0;
}
:host('.has-shadow')  ion-fab-button:not('[disabled]') {
 --box-shadow: 0 10px 20px -10px #{get-color('kirby-primary-shade')};
}
```

File: src/kirby/components/floating-action-button/floating-action-button.component.scss [hunk 4]
```diff
@@ -9,13 +9,12 @@
...
-    }
 }
 
 .no-shadow {
     --box-shadow: 0;
-}
\ No newline at end of file
+}
+
+.disabled {
```
Comment (jakobe) [inline src/kirby/components/floating-action-button/floating-action-button.component.scss]:
```suggestion
ion-fab-button[disabled] {
```

File: src/kirby/components/icon/icon.component.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+:host {
+    display: flex;
```
Comment (jakobe) [inline src/kirby/components/icon/icon.component.scss]: Consider adding explicit rules regarding alignment and distribution...

File: src/app/examples/floating-action-button-example/floating-action-button-example.component.html [hunk 6]
```diff
@@ -13,7 +13,3 @@
 <kirby-floating-action-button>
     <kirby-icon name="close"></kirby-icon>
 </kirby-floating-action-button>
```
Comment (jakobe) [inline src/app/examples/floating-action-button-example/floating-action-button-example.component.html]: Please also add example with `colortype="secondary"`

File: src/app/examples/floating-action-button-example/floating-action-button-example.component.html [hunk 7]
```diff
@@ -13,7 +13,3 @@
 <kirby-floating-action-button>
```
Comment (jakobe) [inline src/app/examples/floating-action-button-example/floating-action-button-example.component.html]: Consider adding comment:
```suggestion
<!-- Change icon: -->
<kirby-floating-action-button>
```

File: src/app/examples/floating-action-button-example/floating-action-button-example.component.html (hunk 6 above)
Comment (szutrifork) [reply src/app/examples/floating-action-button-example/floating-action-button-example.component.html]: we don't have possibility to change FAB color in current version

File: src/kirby/components/floating-action-button/floating-action-button.component.html (hunk 3 above)
Comment (szutrifork) [reply src/kirby/components/floating-action-button/floating-action-button.component.html]: Could't get it to work with HostBinding  but it was simplified in other way

File: src/kirby/components/icon/icon.component.scss [hunk 8]
```diff
@@ -0,0 +1,5 @@
+:host {
+	display: flex;
```
Comment (jakobe) [inline src/kirby/components/icon/icon.component.scss]: Fix indentation:
```suggestion
    display: flex;
```

**Pull request merged**